ssl/t1_trce.c

```diff
@@ -1310,7 +1310,7 @@ static int ssl_print_certificate(BIO *bio, const SSL_CONNECTION *sc, int indent,
         x = NULL;
     }
     if (x ==  NULL)
-        BIO_puts(bio, "<UNPARSEABLE CERTIFICATE>\n");
+        BIO_puts(bio, "<UNPARSABLE CERTIFICATE>\n");
     else {
```

Comment:
nit, the style checker caught this, which isn't your doing (save for the fact that you changed lines where the error occured), but the if clause to this else is supposed to have brackets around it